  INFO root [0x00001c8c] [2018-10-29 23:19:38,389] - 
  INFO root [0x00001c8c] [2018-10-29 23:19:38,477] - -----------------------------------------------------------------------------------------


  INFO root [0x00001c8c] [2018-10-29 23:19:39,972] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001c8c] [2018-10-29 23:19:39,973] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001c8c] [2018-10-29 23:19:39,974] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001c8c] [2018-10-29 23:19:40,467] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x00001c8c] [2018-10-29 23:19:40,521] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x00001c8c] [2018-10-29 23:19:40,525] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:62870 )
  INFO root [0x00001c8c] [2018-10-29 23:19:40,526] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:62870, SOMAXCONN=128.
  INFO root [0x00001c8c] [2018-10-29 23:19:40,527] - server-configs:
  INFO root [0x00001c8c] [2018-10-29 23:19:40,527] - 	gameUpdateHertz : 10
  INFO root [0x00001c8c] [2018-10-29 23:19:40,527] - 	entryScriptFile : kbemain
  INFO root [0x00001c8c] [2018-10-29 23:19:40,528] - 	internalAddr : 192.168.0.107:62870
  INFO root [0x00001c8c] [2018-10-29 23:19:40,528] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x00001c8c] [2018-10-29 23:19:40,528] - 	externalCustomAddr : 127.0.0.1
  INFO root [0x00001c8c] [2018-10-29 23:19:40,529] - 	componentID : 7001
  INFO root [0x00001c8c] [2018-10-29 23:19:41,473] - Components::findLogger: found logger, addr:192.168.0.107:62849
  INFO root [0x00001c8c] [2018-10-29 23:19:41,473] - Components::addComponent: reset orderLog, uid=285!
  INFO root [0x00001c8c] [2018-10-29 23:19:41,474] - Components::addComponent[logger], uid=285, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001c8c] [2018-10-29 23:19:41,613] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 285. PID: 7340 ----
  INFO root [0x00001c8c] [2018-10-29 23:19:41,613] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001c8c] [2018-10-29 23:19:41,640] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001c8c] [2018-10-29 23:19:41,751] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001c8c] [2018-10-29 23:19:45,634] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001c8c] [2018-10-29 23:19:45,717] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001c8c] [2018-10-29 23:19:45,856] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001c8c] [2018-10-29 23:19:46,701] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001c8c] [2018-10-29 23:19:46,797] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001c8c] [2018-10-29 23:19:46,855] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001c8c] [2018-10-29 23:19:46,921] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001c8c] [2018-10-29 23:19:47,301] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001c8c] [2018-10-29 23:19:47,991] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001c8c] [2018-10-29 23:19:47,992] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001c8c] [2018-10-29 23:19:48,968] - loaded entity-script:Account(1).
 DEBUG root [0x00001c8c] [2018-10-29 23:19:48,970] - loaded entity-script:CheckName(2).
 DEBUG root [0x00001c8c] [2018-10-29 23:19:49,044] - loaded entity-script:Halls(3).
 DEBUG root [0x00001c8c] [2018-10-29 23:19:49,072] - loaded entity-script:Room(4).
  INFO root [0x00001c8c] [2018-10-29 23:19:49,073] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001c8c] [2018-10-29 23:19:49,172] - TelnetServer server is running on port 40000
  INFO root [0x00001c8c] [2018-10-29 23:19:49,627] - ---- baseapp is running ----
 DEBUG root [0x00001c8c] [2018-10-29 23:19:49,627] - Components::process(): Request for the process of identity...
  WARN root [0x00001c8c] [2018-10-29 23:19:55,817] - Forwarding logs to logger[192.168.0.107:62849/0/0/0]...
 ERROR root [0x00001c8c] [2018-10-29 23:36:02,163] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001c8c] [2018-10-29 23:36:02,164] - The following logs sent to logger failed:
  WARN root [0x00001c8c] [2018-10-29 23:36:02,165] - ==> [2018-10-29 23:36:02 163] TCPPacketReceiver::processPendingEvents(192.168.0.107:62849): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001c8c] [2018-10-29 23:36:02,165] - ==> [2018-10-29 23:36:02 164] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001c8c] [2018-10-29 23:36:02,209] - TCPPacketReceiver::processPendingEvents(192.168.0.107:62851): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001c8c] [2018-10-29 23:36:02,210] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001c8c] [2018-10-29 23:36:02,233] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001c8c] [2018-10-29 23:36:02,233] - onBaseAppShutDown: state=0
  INFO root [0x00001c8c] [2018-10-29 23:36:02,233] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001c8c] [2018-10-29 23:36:02,233] - onBaseAppShutDown: state=1
  INFO root [0x00001c8c] [2018-10-29 23:36:02,233] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001c8c] [2018-10-29 23:36:02,234] - onBaseAppShutDown: state=2
 DEBUG root [0x00001c8c] [2018-10-29 23:36:02,329] - Halls::~Halls(): 2001
  INFO root [0x00000fac] [2018-10-29 23:36:03,282] - 
  INFO root [0x00000fac] [2018-10-29 23:36:03,284] - -----------------------------------------------------------------------------------------


  INFO root [0x00000fac] [2018-10-29 23:36:03,285] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00000fac] [2018-10-29 23:36:03,285] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00000fac] [2018-10-29 23:36:03,285] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00000fac] [2018-10-29 23:36:03,292] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x00000fac] [2018-10-29 23:36:03,292] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x00000fac] [2018-10-29 23:36:03,294] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:63156 )
  INFO root [0x00000fac] [2018-10-29 23:36:03,294] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:63156, SOMAXCONN=128.
  INFO root [0x00000fac] [2018-10-29 23:36:03,294] - server-configs:
  INFO root [0x00000fac] [2018-10-29 23:36:03,294] - 	gameUpdateHertz : 10
  INFO root [0x00000fac] [2018-10-29 23:36:03,294] - 	entryScriptFile : kbemain
  INFO root [0x00000fac] [2018-10-29 23:36:03,295] - 	internalAddr : 192.168.0.107:63156
  INFO root [0x00000fac] [2018-10-29 23:36:03,295] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x00000fac] [2018-10-29 23:36:03,295] - 	externalCustomAddr : 192.168.0.107
  INFO root [0x00000fac] [2018-10-29 23:36:03,295] - 	componentID : 7001
  INFO root [0x00000fac] [2018-10-29 23:36:04,968] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 3619. PID: 7212 ----
  INFO root [0x00000fac] [2018-10-29 23:36:04,968] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00000fac] [2018-10-29 23:36:04,968] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00000fac] [2018-10-29 23:36:04,969] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,092] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,092] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,093] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,096] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,096] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,096] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,097] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,123] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00000fac] [2018-10-29 23:36:05,155] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00000fac] [2018-10-29 23:36:05,155] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,175] - loaded entity-script:Account(1).
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,176] - loaded entity-script:CheckName(2).
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,181] - loaded entity-script:Halls(3).
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,182] - loaded entity-script:Room(4).
  INFO root [0x00000fac] [2018-10-29 23:36:05,183] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00000fac] [2018-10-29 23:36:05,183] - TelnetServer server is running on port 40000
  INFO root [0x00000fac] [2018-10-29 23:36:05,474] - ---- baseapp is running ----
 DEBUG root [0x00000fac] [2018-10-29 23:36:05,474] - Components::process(): Request for the process of identity...
  WARN root [0x00000fac] [2018-10-29 23:36:05,784] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20472, errorno=10048
  WARN root [0x00000fac] [2018-10-29 23:36:09,128] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20372, errorno=10048
  WARN root [0x00000fac] [2018-10-29 23:36:12,191] - EntityApp::checkTickPeriod: tick took 6717% (6.72 seconds)!
  INFO root [0x00000fac] [2018-10-29 23:36:13,274] - Components::findComponents: find logger(1)...
  INFO root [0x00000fac] [2018-10-29 23:36:13,353] - Components::findComponents: found logger, addr:192.168.0.107:63160
  INFO root [0x00000fac] [2018-10-29 23:36:13,354] - Components::addComponent: reset orderLog, uid=3619!
  INFO root [0x00000fac] [2018-10-29 23:36:13,354] - Components::addComponent[logger], uid=3619, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000fac] [2018-10-29 23:36:13,356] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00000fac] [2018-10-29 23:36:13,457] - Components::findComponents: found dbmgr, addr:192.168.0.107:63155
  INFO root [0x00000fac] [2018-10-29 23:36:13,458] - Components::addComponent: reset orderLog, uid=3619!
  INFO root [0x00000fac] [2018-10-29 23:36:13,458] - Components::addComponent[dbmgr], uid=3619, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00000fac] [2018-10-29 23:36:13,459] - Components::findComponents: found dbmgr, addr:192.168.0.107:63155
  WARN root [0x00000fac] [2018-10-29 23:36:14,460] - EntityApp::checkTickPeriod: tick took 1285% (1.28 seconds)!
  WARN root [0x00000fac] [2018-10-29 23:36:14,462] - Forwarding logs to logger[192.168.0.107:63160/0/0/0]...
 ERROR root [0x00000fac] [2018-10-29 23:38:15,629] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00000fac] [2018-10-29 23:38:15,633] - The following logs sent to logger failed:
  WARN root [0x00000fac] [2018-10-29 23:38:15,634] - ==> [2018-10-29 23:38:15 628] TCPPacketReceiver::processPendingEvents(192.168.0.107:63160): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000fac] [2018-10-29 23:38:15,634] - ==> [2018-10-29 23:38:15 629] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00000fac] [2018-10-29 23:38:15,674] - TCPPacketReceiver::processPendingEvents(192.168.0.107:63155): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00000fac] [2018-10-29 23:38:15,675] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00000fac] [2018-10-29 23:38:15,678] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00000fac] [2018-10-29 23:38:15,680] - onBaseAppShutDown: state=0
  INFO root [0x00000fac] [2018-10-29 23:38:15,680] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00000fac] [2018-10-29 23:38:15,680] - onBaseAppShutDown: state=1
  INFO root [0x00000fac] [2018-10-29 23:38:15,680] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00000fac] [2018-10-29 23:38:15,680] - onBaseAppShutDown: state=2
 ERROR root [0x00000fac] [2018-10-29 23:38:15,707] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00000fac] [2018-10-29 23:38:15,708] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,708] - Account::~Account(): 2002
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,714] - Halls::~Halls(): 2001
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,715] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00000fac] [2018-10-29 23:38:15,727] - Script::uninstall(): is successfully!
 DEBUG root [0x00000fac] [2018-10-29 23:38:15,727] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000005c8] [2018-10-29 23:38:16,724] - 
  INFO root [0x000005c8] [2018-10-29 23:38:16,743] - -----------------------------------------------------------------------------------------


  INFO root [0x000005c8] [2018-10-29 23:38:16,743] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000005c8] [2018-10-29 23:38:16,744] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000005c8] [2018-10-29 23:38:16,744] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000005c8] [2018-10-29 23:38:16,747] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x000005c8] [2018-10-29 23:38:16,747] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:63205 )
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:63205, SOMAXCONN=128.
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - server-configs:
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - 	gameUpdateHertz : 10
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - 	entryScriptFile : kbemain
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - 	internalAddr : 192.168.0.107:63205
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x000005c8] [2018-10-29 23:38:16,748] - 	externalCustomAddr : 171.88.53.16
  INFO root [0x000005c8] [2018-10-29 23:38:16,749] - 	componentID : 7001
  WARN root [0x000005c8] [2018-10-29 23:38:17,256] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20787, errorno=10048
  INFO root [0x000005c8] [2018-10-29 23:38:17,260] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 4053. PID: 7636 ----
  INFO root [0x000005c8] [2018-10-29 23:38:17,260] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000005c8] [2018-10-29 23:38:17,260] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000005c8] [2018-10-29 23:38:17,261] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,384] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,384] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,384] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,387] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,387] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,388] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,388] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,414] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000005c8] [2018-10-29 23:38:17,440] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000005c8] [2018-10-29 23:38:17,440] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,460] - loaded entity-script:Account(1).
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,461] - loaded entity-script:CheckName(2).
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,466] - loaded entity-script:Halls(3).
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,467] - loaded entity-script:Room(4).
  INFO root [0x000005c8] [2018-10-29 23:38:17,467] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000005c8] [2018-10-29 23:38:17,468] - TelnetServer server is running on port 40000
  INFO root [0x000005c8] [2018-10-29 23:38:17,741] - ---- baseapp is running ----
 DEBUG root [0x000005c8] [2018-10-29 23:38:17,742] - Components::process(): Request for the process of identity...
  WARN root [0x000005c8] [2018-10-29 23:38:23,881] - EntityApp::checkTickPeriod: tick took 6139% (6.14 seconds)!
  INFO root [0x000005c8] [2018-10-29 23:38:24,942] - Components::findComponents: find logger(1)...
  INFO root [0x000005c8] [2018-10-29 23:38:25,026] - Components::findComponents: found logger, addr:192.168.0.107:63206
  INFO root [0x000005c8] [2018-10-29 23:38:25,027] - Components::addComponent: reset orderLog, uid=4053!
  INFO root [0x000005c8] [2018-10-29 23:38:25,027] - Components::addComponent[logger], uid=4053, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000005c8] [2018-10-29 23:38:25,029] - Components::findComponents: find dbmgr(2)...
  INFO root [0x000005c8] [2018-10-29 23:38:25,130] - Components::findComponents: found dbmgr, addr:192.168.0.107:63210
  INFO root [0x000005c8] [2018-10-29 23:38:25,131] - Components::addComponent: reset orderLog, uid=4053!
  INFO root [0x000005c8] [2018-10-29 23:38:25,131] - Components::addComponent[dbmgr], uid=4053, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000005c8] [2018-10-29 23:38:25,132] - Components::findComponents: found dbmgr, addr:192.168.0.107:63210
  WARN root [0x000005c8] [2018-10-29 23:38:26,132] - EntityApp::checkTickPeriod: tick took 1290% (1.29 seconds)!
  WARN root [0x000005c8] [2018-10-29 23:38:26,134] - Forwarding logs to logger[192.168.0.107:63206/0/0/0]...
 ERROR root [0x000005c8] [2018-10-29 23:42:16,739] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000005c8] [2018-10-29 23:42:16,744] - The following logs sent to logger failed:
  WARN root [0x000005c8] [2018-10-29 23:42:16,745] - ==> [2018-10-29 23:42:16 737] TCPPacketReceiver::processPendingEvents(192.168.0.107:63206): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000005c8] [2018-10-29 23:42:16,745] - ==> [2018-10-29 23:42:16 739] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000005c8] [2018-10-29 23:42:16,786] - TCPPacketReceiver::processPendingEvents(192.168.0.107:63210): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000005c8] [2018-10-29 23:42:16,788] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000005c8] [2018-10-29 23:42:16,791] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000005c8] [2018-10-29 23:42:16,791] - onBaseAppShutDown: state=0
  INFO root [0x000005c8] [2018-10-29 23:42:16,792] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000005c8] [2018-10-29 23:42:16,792] - onBaseAppShutDown: state=1
  INFO root [0x000005c8] [2018-10-29 23:42:16,792] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000005c8] [2018-10-29 23:42:16,792] - onBaseAppShutDown: state=2
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,800] - Halls::~Halls(): 2001
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,800] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,801] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000005c8] [2018-10-29 23:42:16,811] - Script::uninstall(): is successfully!
 DEBUG root [0x000005c8] [2018-10-29 23:42:16,811] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001360] [2018-10-29 23:42:17,852] - 
  INFO root [0x00001360] [2018-10-29 23:42:17,852] - -----------------------------------------------------------------------------------------


  INFO root [0x00001360] [2018-10-29 23:42:17,853] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001360] [2018-10-29 23:42:17,853] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001360] [2018-10-29 23:42:17,853] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001360] [2018-10-29 23:42:17,857] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x00001360] [2018-10-29 23:42:17,857] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x00001360] [2018-10-29 23:42:17,859] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:63309 )
  INFO root [0x00001360] [2018-10-29 23:42:17,859] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:63309, SOMAXCONN=128.
  INFO root [0x00001360] [2018-10-29 23:42:17,859] - server-configs:
  INFO root [0x00001360] [2018-10-29 23:42:17,859] - 	gameUpdateHertz : 10
  INFO root [0x00001360] [2018-10-29 23:42:17,859] - 	entryScriptFile : kbemain
  INFO root [0x00001360] [2018-10-29 23:42:17,860] - 	internalAddr : 192.168.0.107:63309
  INFO root [0x00001360] [2018-10-29 23:42:17,860] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x00001360] [2018-10-29 23:42:17,860] - 	externalCustomAddr : 171.88.53.16
  INFO root [0x00001360] [2018-10-29 23:42:17,860] - 	componentID : 7001
  INFO root [0x00001360] [2018-10-29 23:42:19,532] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 4840. PID: 7452 ----
  INFO root [0x00001360] [2018-10-29 23:42:19,533] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001360] [2018-10-29 23:42:19,533] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001360] [2018-10-29 23:42:19,534] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001360] [2018-10-29 23:42:19,657] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001360] [2018-10-29 23:42:19,658] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001360] [2018-10-29 23:42:19,658] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001360] [2018-10-29 23:42:19,661] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001360] [2018-10-29 23:42:19,661] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001360] [2018-10-29 23:42:19,661] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001360] [2018-10-29 23:42:19,662] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001360] [2018-10-29 23:42:19,688] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001360] [2018-10-29 23:42:19,715] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001360] [2018-10-29 23:42:19,715] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001360] [2018-10-29 23:42:19,735] - loaded entity-script:Account(1).
 DEBUG root [0x00001360] [2018-10-29 23:42:19,736] - loaded entity-script:CheckName(2).
 DEBUG root [0x00001360] [2018-10-29 23:42:19,741] - loaded entity-script:Halls(3).
 DEBUG root [0x00001360] [2018-10-29 23:42:19,742] - loaded entity-script:Room(4).
  INFO root [0x00001360] [2018-10-29 23:42:19,742] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001360] [2018-10-29 23:42:19,742] - TelnetServer server is running on port 40000
  INFO root [0x00001360] [2018-10-29 23:42:20,015] - ---- baseapp is running ----
 DEBUG root [0x00001360] [2018-10-29 23:42:20,016] - Components::process(): Request for the process of identity...
  INFO root [0x000011c0] [2018-10-29 23:42:25,186] - 
  INFO root [0x000011c0] [2018-10-29 23:42:25,186] - -----------------------------------------------------------------------------------------


  INFO root [0x000011c0] [2018-10-29 23:42:25,187] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000011c0] [2018-10-29 23:42:25,187] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000011c0] [2018-10-29 23:42:25,187] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000011c0] [2018-10-29 23:42:25,189] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x000011c0] [2018-10-29 23:42:25,189] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:63326 )
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:63326, SOMAXCONN=128.
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - server-configs:
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - 	gameUpdateHertz : 10
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - 	entryScriptFile : kbemain
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - 	internalAddr : 192.168.0.107:63326
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x000011c0] [2018-10-29 23:42:25,191] - 	externalCustomAddr : 171.88.53.16
  INFO root [0x000011c0] [2018-10-29 23:42:25,192] - 	componentID : 7001
  INFO root [0x000011c0] [2018-10-29 23:42:26,873] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 4863. PID: 3636 ----
  INFO root [0x000011c0] [2018-10-29 23:42:26,874] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000011c0] [2018-10-29 23:42:26,874] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000011c0] [2018-10-29 23:42:26,874] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000011c0] [2018-10-29 23:42:26,998] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000011c0] [2018-10-29 23:42:26,998] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000011c0] [2018-10-29 23:42:26,998] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,001] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,002] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,002] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,002] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,030] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000011c0] [2018-10-29 23:42:27,058] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000011c0] [2018-10-29 23:42:27,058] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,078] - loaded entity-script:Account(1).
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,079] - loaded entity-script:CheckName(2).
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,084] - loaded entity-script:Halls(3).
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,085] - loaded entity-script:Room(4).
  INFO root [0x000011c0] [2018-10-29 23:42:27,085] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000011c0] [2018-10-29 23:42:27,085] - TelnetServer server is running on port 40000
  INFO root [0x000011c0] [2018-10-29 23:42:27,360] - ---- baseapp is running ----
 DEBUG root [0x000011c0] [2018-10-29 23:42:27,360] - Components::process(): Request for the process of identity...
  WARN root [0x000011c0] [2018-10-29 23:42:33,448] - EntityApp::checkTickPeriod: tick took 6088% (6.09 seconds)!
  INFO root [0x000011c0] [2018-10-29 23:42:34,488] - Components::findComponents: find logger(1)...
  INFO root [0x000011c0] [2018-10-29 23:42:34,524] - Components::findComponents: found logger, addr:192.168.0.107:63325
  INFO root [0x000011c0] [2018-10-29 23:42:34,524] - Components::addComponent: reset orderLog, uid=4863!
  INFO root [0x000011c0] [2018-10-29 23:42:34,525] - Components::addComponent[logger], uid=4863, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000011c0] [2018-10-29 23:42:34,527] - Components::findComponents: find dbmgr(2)...
  INFO root [0x000011c0] [2018-10-29 23:42:34,628] - Components::findComponents: found dbmgr, addr:192.168.0.107:63320
  INFO root [0x000011c0] [2018-10-29 23:42:34,629] - Components::addComponent: reset orderLog, uid=4863!
  INFO root [0x000011c0] [2018-10-29 23:42:34,630] - Components::addComponent[dbmgr], uid=4863, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000011c0] [2018-10-29 23:42:34,630] - Components::findComponents: found dbmgr, addr:192.168.0.107:63320
  WARN root [0x000011c0] [2018-10-29 23:42:35,630] - Forwarding logs to logger[192.168.0.107:63325/0/0/0]...
 ERROR root [0x000011c0] [2018-10-29 23:42:41,317] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000011c0] [2018-10-29 23:42:41,325] - The following logs sent to logger failed:
  WARN root [0x000011c0] [2018-10-29 23:42:41,325] - ==> [2018-10-29 23:42:41 315] TCPPacketReceiver::processPendingEvents(192.168.0.107:63325): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000011c0] [2018-10-29 23:42:41,325] - ==> [2018-10-29 23:42:41 317] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000011c0] [2018-10-29 23:42:41,365] - TCPPacketReceiver::processPendingEvents(192.168.0.107:63320): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000011c0] [2018-10-29 23:42:41,367] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000011c0] [2018-10-29 23:42:41,373] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000011c0] [2018-10-29 23:42:41,374] - onBaseAppShutDown: state=0
  INFO root [0x000011c0] [2018-10-29 23:42:41,374] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000011c0] [2018-10-29 23:42:41,375] - onBaseAppShutDown: state=1
  INFO root [0x000011c0] [2018-10-29 23:42:41,375] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000011c0] [2018-10-29 23:42:41,375] - onBaseAppShutDown: state=2
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,383] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,383] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,384] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x000011c0] [2018-10-29 23:42:41,394] - Script::uninstall(): is successfully!
 DEBUG root [0x000011c0] [2018-10-29 23:42:41,394] - ThreadPool::destroy(): starting size 2.
  INFO root [0x000026a0] [2018-10-29 23:55:58,955] - 
  INFO root [0x000026a0] [2018-10-29 23:55:58,956] - -----------------------------------------------------------------------------------------


  INFO root [0x000026a0] [2018-10-29 23:55:58,957] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000026a0] [2018-10-29 23:55:58,957] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000026a0] [2018-10-29 23:55:58,957] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000026a0] [2018-10-29 23:55:58,963] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:20015 )
  INFO root [0x000026a0] [2018-10-29 23:55:58,963] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.107:20015, SOMAXCONN=128.
  INFO root [0x000026a0] [2018-10-29 23:55:58,965] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.107 ( 192.168.0.107:59044 )
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - NetworkInterface::initialize(INTERNAL): address 192.168.0.107:59044, SOMAXCONN=128.
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - server-configs:
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - 	gameUpdateHertz : 10
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - 	entryScriptFile : kbemain
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - 	internalAddr : 192.168.0.107:59044
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - 	externalAddr : 192.168.0.107:20015
  INFO root [0x000026a0] [2018-10-29 23:55:58,966] - 	externalCustomAddr : 192.168.0.107
  INFO root [0x000026a0] [2018-10-29 23:55:58,967] - 	componentID : 7001
  INFO root [0x000026a0] [2018-10-29 23:56:00,659] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 7521. PID: 6852 ----
  INFO root [0x000026a0] [2018-10-29 23:56:00,660] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000026a0] [2018-10-29 23:56:00,660] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000026a0] [2018-10-29 23:56:00,660] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,782] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,782] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,782] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,785] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,786] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,786] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,786] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,812] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000026a0] [2018-10-29 23:56:00,839] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000026a0] [2018-10-29 23:56:00,839] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,860] - loaded entity-script:Account(1).
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,861] - loaded entity-script:CheckName(2).
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,866] - loaded entity-script:Halls(3).
 DEBUG root [0x000026a0] [2018-10-29 23:56:00,867] - loaded entity-script:Room(4).
  INFO root [0x000026a0] [2018-10-29 23:56:00,868] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000026a0] [2018-10-29 23:56:00,868] - TelnetServer server is running on port 40000
  INFO root [0x000026a0] [2018-10-29 23:56:01,142] - ---- baseapp is running ----
 DEBUG root [0x000026a0] [2018-10-29 23:56:01,143] - Components::process(): Request for the process of identity...
  WARN root [0x000026a0] [2018-10-29 23:56:07,249] - EntityApp::checkTickPeriod: tick took 6106% (6.11 seconds)!
  INFO root [0x000026a0] [2018-10-29 23:56:08,344] - Components::findComponents: find logger(1)...
  INFO root [0x000026a0] [2018-10-29 23:56:08,366] - Components::findComponents: found logger, addr:192.168.0.107:59041
  INFO root [0x000026a0] [2018-10-29 23:56:08,367] - Components::addComponent: reset orderLog, uid=7521!
  INFO root [0x000026a0] [2018-10-29 23:56:08,367] - Components::addComponent[logger], uid=7521, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000026a0] [2018-10-29 23:56:08,368] - Components::findComponents: find dbmgr(2)...
  INFO root [0x000026a0] [2018-10-29 23:56:08,474] - Components::findComponents: found dbmgr, addr:192.168.0.107:59043
  INFO root [0x000026a0] [2018-10-29 23:56:08,474] - Components::addComponent: reset orderLog, uid=7521!
  INFO root [0x000026a0] [2018-10-29 23:56:08,475] - Components::addComponent[dbmgr], uid=7521, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000026a0] [2018-10-29 23:56:08,476] - Components::findComponents: found dbmgr, addr:192.168.0.107:59043
  WARN root [0x000026a0] [2018-10-29 23:56:09,476] - EntityApp::checkTickPeriod: tick took 1232% (1.23 seconds)!
  WARN root [0x000026a0] [2018-10-29 23:56:09,478] - Forwarding logs to logger[192.168.0.107:59041/0/0/0]...
